src/hooks/contextValidator.ts

```diff
@@ -31,7 +31,11 @@ export function contextNotEmptyValidator(
 export const contextOptionClientTokenNotEmptyValidator = (
     scriptContext: ScriptContextState | null
 ): StrictScriptContextState => {
-    if (scriptContext?.options[DATA_CLIENT_TOKEN] == null) {
+    if (
```

Comment:
Can we simplify this case with just using truthy checks?
```suggestion
    if (scriptContext && scriptContext.options[DATA_CLIENT_TOKEN])
```